wazo_confd/helpers/destination.py

```diff
@@ -417,6 +417,34 @@ def make_user_fields_flat(self, data):
         data.pop('user', None)
         return data
 
+    @pre_dump
+    def separate_action(self, data):
+        options = data.actionarg2.split(';') if data.actionarg2 else []
+        data.ring_time = None
+        data.moh_uuid = None
+
+        if len(options) > 0:
+            data.ring_time = options[0] or None
+
+        if len(options) > 1:  # id is always bound with variables
+            data.moh_uuid = options[1]
+
+        return data
+
+    @post_load
+    def merge_action(self, data):
+        ring_time = data.pop('ring_time', None)
+        moh_uuid = data.pop('moh_uuid', None)
+
+        actionarg2 = ''
+        if ring_time:
```

Comment:
What if `ring_time` is 0? Is that a use case?

Reply:
You are right, fixed!